src/Microsoft.ComponentDetection.Orchestrator/Orchestrator.cs

```diff
@@ -265,18 +277,16 @@ private async Task<ScanResult> SafelyExecuteAsync(BcdeExecutionTelemetryRecord r
             else
             {
                 // On an exception, return error to dotnet core
-                this.logger.LogError($"There was an unexpected error: ");
-                this.logger.LogException(e, isError: true);
+                this.logger.LogError(e, "There was an unexpected error");
 
                 var errorMessage = new StringBuilder();
                 errorMessage.AppendLine(e.ToString());
                 if (e is ReflectionTypeLoadException refEx && refEx.LoaderExceptions != null)
                 {
                     foreach (var loaderException in refEx.LoaderExceptions)
                     {
-                        var loaderExceptionString = loaderException.ToString();
-                        this.logger.LogError(loaderExceptionString);
-                        errorMessage.AppendLine(loaderExceptionString);
+                        this.logger.LogError(loaderException, "Got exception");
```

Comment:
:joy: is the text mandatory?

Reply:
I am not sure this exception can happen anymore since the migration to DI, but that's a separate issue.

Reply:
Yes! 😆